pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -272,21 +272,6 @@ public void runSegmentMetrics() {
           nErrors);
       _metricsRegistry.setValueOfTableGauge(tableName, ControllerGauge.PERCENT_SEGMENTS_AVAILABLE,
           (nSegments > 0) ? (100 - (nOffline * 100 / nSegments)) : 100);
-
-      // Emit estimated sizes of realtime/offline tables.
-      TableSizeReader.TableSizeDetails tableSizeDetails = _tableSizeReader.getTableSizeDetails(tableName,
-          _config.getServerAdminRequestTimeoutSeconds() * 1000);
-      if (tableSizeDetails != null) {
-        if (tableSizeDetails.realtimeSegments != null) {
```

Comment:
Do consuming segments count towards realtime segment size?

Reply:
Not sure. For now, we are backing out emitting the metric for realtime tables.